azure_pipelines/automation.yml

```diff
@@ -29,82 +29,14 @@ pool:
 
 jobs:
 - job: ui_automation
-  displayName: Run MSAL UI automation
+  displayName: 'Run MSAL UI automation'
   timeoutInMinutes: 360
   cancelTimeoutInMinutes: 5
   workspace:
     clean: all
-  
-  steps:
-
-  - checkout: self
```

Comment:
So all of this code was moved into `templates/tests-with-conf-file.yml` so it can be parametrised with parameters.schema for both?

Reply:
Correct. This template is used for MSAL Objective-C UI tests and Native auth E2E tests. I used the MSAL Objective-C UI tests as default value for these parameters.